index.html

```diff
@@ -217,6 +217,37 @@ <h6>LET'S CONNECT</h6>
             </li>
         </ul>
     </section>
+    <footer class="contact-form">
```

Comment:
- Hey, please, note that this should be a `section` and not a `footer` as this design does not have a footer. please, update this to a section to enforce a  better use of the html5 semantic usage on your web page.